tasks-by-users/Naufil-Shemle/05-card-hover-effect/style.css

```diff
@@ -0,0 +1,135 @@
+body {
+  margin: 0;
+  padding: 0;
+
+  min-height: 100vh;
+
+  background: #333;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+img {
+  width: 128px;
+  filter: invert(100%);
```

Comment:
What does this Invert property do? @Naufil-Shemle

Reply:
Icon was in black color so I wanna changes its color to white that's why I used this property